evm.md

```diff
@@ -2442,6 +2449,13 @@ A `ScheduleConst` is a constant determined by the fee schedule.
 ```k
     syntax Schedule ::= "ISTANBUL" [klabel(ISTANBUL_EVM), symbol]
  // -------------------------------------------------------------
+    rule Gsload        < ISTANBUL > => 800
+    rule Gsstoreset    < ISTANBUL > => 20000
+    rule Gsstorereset  < ISTANBUL > => 5000
+    rule Rsstoreclear  < ISTANBUL > => 15000
+    rule Gstipend      < ISTANBUL > => 2300
+    rule Ghasdirtysstore << ISTANBUL >> => false
+
     rule SCHEDCONST < ISTANBUL > => SCHEDCONST < PETERSBURG >
```

Comment:
Add side conditions to these fallback rules which say that we only use `PETERSBURG` if the `SCHEDCONST` is not the same as one of the ones you've modified above.